pkg/common/ingress/controller/backend_ssl.go

```diff
@@ -111,7 +111,13 @@ func (ic *GenericController) getPemCertificate(secret *apiv1.Secret) (*ingress.S
 		}
 
 	} else if ca != nil {
-		s, err = ssl.AddCertAuth(nsSecName, ca)
+		crl := []byte{}
+		_, crlok := secret.Data["ca.crl"]
+		if crlok {
+			crl = secret.Data["ca.crl"]
+		}
+
```

Comment:
Just a `crl, _ := secret.Data["ca.crl"]` should be enough.

Reply:
Yeah, not sure what was going through my mind when I wrote this!  will amend this into the commit.